examples/experimental/dagster-airlift/dagster_airlift/within_airflow.py

```diff
@@ -0,0 +1,61 @@
+import json
+import sys
+
+from airflow import DAG
+
+
+def mark_as_dagster_migrating(
```

Comment:
I think we should have a separate inner function here that has nothing to do with `globals` shenanigans and just takes a DAG and returns a DAG so that can be independently tested